UI/window-basic-main.cpp

```diff
@@ -1753,6 +1753,8 @@ void OBSBasic::OBSInit()
 			QByteArray::fromBase64(QByteArray(dockStateStr));
 		if (!restoreState(dockState))
 			on_resetUI_triggered();
+
+		activateWindow();
```

Comment:
Is this call required? `on_resetUI_triggered` also calls `activateWindow()`

Reply:
`on_resetUI_triggered()` is only called when the call to `restoreState(...)` fails. Therefore the call is required in case it succeeds. 

Putting the `activateWindow()` function in the else block should avoid calling it twice.

Reply:
However I removed this line in the fix and added a call to `activateWindow()` right after the main window loaded. This addresses the original bug and also the bug as described [here](https://github.com/obsproject/obs-studio/issues/2209#issuecomment-630308831)